shell.nix

```diff
@@ -0,0 +1,65 @@
+{ nixroot  ? (import <nixpkgs> {}) }:
+let inherit (nixroot) stdenv
+    chromaprint fftw flac libid3tag libmad libopus libshout libsndfile
+    libusb1 libvorbis libebur128 pkgconfig portaudio portmidi protobuf qt4 qt5
+    rubberband scons sqlite taglib soundtouch vamp opusfile hidapi upower ccache git;
+  
+in stdenv.mkDerivation rec {
+  name = "mixxx-${version}";
+  version = "2.1.0-dev";
```

Comment:
isn't it possible to grab version from source instead of hard-coding it here ?

Reply:
I guess. there are some nix functions to readout stuff from filesystem.

Reply:
bump version to 2.2.0-dev